### ISSUE TITLE
MetaData does not need to be loaded and persisted every time for cluster mode

### ISSUE THREAD

**Issue by menghaoranss** [apache/shardingsphere#11958]
Method `init` in `ClusterContextManager.java`

```
public void init(final MetaDataContexts metaDataContexts, final TransactionContexts transactionContexts) {
        this.metaDataContexts = metaDataContexts;
        this.transactionContexts = transactionContexts;
        ShardingSphereEventBus.getInstance().register(this);
        disableDataSources();
        persistMetaData();
        lock = createShardingSphereLock(registryCenter.getRepository());
        registryCenter.onlineInstance(metaDataContexts.getAllSchemaNames());
    }
```

If `overwrite` is configured as `false` MetaData should be loaded from registry center.

**Comment by soulasuna** [comment]
The configuration file does not contain `TableMetaData` which should be invisible to the user. So it should not be controlled by `overwrite` to load or not.
The changed `TableMetaData` is synchronized and broadcasted in the cluster when each proxy starts in the same cluster.
@menghaoranss Please assign to me if you agree.

**Comment by menghaoranss** [comment]
> The configuration file does not contain `TableMetaData` which should be invisible to the user. So it should not be controlled by `overwrite` to load or not.
> The changed `TableMetaData` is synchronized and broadcasted in the cluster when each proxy starts in the same cluster.
> @menghaoranss Please assign to me if you agree.

@soulasuna cool, I agree with you

**Comment by menghaoranss** [comment]
fixed by #15875
